aim/cli/upgrade/utils.py

```diff
@@ -47,12 +47,12 @@ def convert_run(lrun: LegacyRun, repo: Repo, legacy_run_map, skip_failed):
         run[...] = lrun.params
         run['v2_params'] = {'run_hash': lrun.run_hash}
```

Comment:
What is `v2_params` exactly? If you mean some legacy params we do not store anymore - maybe use 'legacy_params'?

Reply:
v2 seems "newer" at the first glance. Also, these params may have come from v1 too.

Reply:
> What is `v2_params` exactly? If you mean some legacy params we do not store anymore - maybe use 'legacy_params'?

good point. @gorarakelyan any objections on renaming to `legacy_params`?